.wordlist.txt

```diff
@@ -217,7 +219,7 @@ MicroSol
 Minecraft
 MineOS
 MiniDLNA
-Minio
+MinIO
```

Comment:
Good find 👍, I'll rename it in dietpi-software as well.

Reply:
thanks. it's a very nice app